src/styles/blocks/catalog.scss

```diff
@@ -0,0 +1,21 @@
+.main {
+  font-weight: 400;
+}
+
+.catalog {
+  box-sizing: border-box;
+  display: flex;
+  flex-wrap: wrap;
+  align-items: flex-start;
+  padding: 50px 0;
+  margin: 0 auto;
+  width: 944px;
```

Comment:
the catalog width should be calculated depending on the number of product cards and their width